stuffer/s2n_stuffer_base64.c

```diff
@@ -139,7 +139,6 @@ int s2n_stuffer_read_base64(struct s2n_stuffer *stuffer, struct s2n_stuffer *out
             /* Put the next two bits in the third data byte */
             /* Put the next six bits in the fourth data byte */
             *ptr = ((value3 << 6) & 0xc0) | (value4 & 0x3f);
-            ptr++;
```

Comment:
Q: Why are we getting rid of the increment here?

Reply:
It doesn't actually do anything. The loop ends right after this line, and `ptr` is re-initialized on every iteration of the loop.

Reply:
If we're modifying this code, it may be better to remove line 129, and switch the order of assignment/increment in the conditionals. That will always remove the increment on 129 even if bytes_this_round is 0 or 1.